clr-installer/main.go

```diff
@@ -445,6 +445,10 @@ func processOptionsSaveIfSet(options args.Args, md *model.SystemInstall) {
 		md.MediaOpts.SwapFileSize = options.SwapFileSize
 		md.MediaOpts.SwapFileSet = true
 	}
+
+	if options.ForceInstall {
```

Comment:
Since the ForceDestructive is also for the media, this should probably be added to the mediaOptions, but not allowed in the YAML.

Reply:
Thats a good point.. will do

Reply:
Done